deploy/swrs/transform/function/load_fuel_charge.sql

```diff
@@ -8,538 +8,126 @@ create or replace function swrs_transform.load_fuel_charge()
+        INSERT INTO swrs_load.fuel_charge (fuel_charge, start_date, end_date, fuel_carbon_tax_details_id, carbon_tax_act_fuel_type_id) VALUES (0.1285, '2018-04-01', '2019-03-31',59, 18);
+        INSERT INTO swrs_load.fuel_charge (fuel_charge, start_date, end_date, fuel_carbon_tax_details_id, carbon_tax_act_fuel_type_id) VALUES (0.1468, '2019-04-01', '2020-03-31',59, 18);
+        INSERT INTO swrs_load.fuel_charge (fuel_charge, start_date, end_date, fuel_carbon_tax_details_id, carbon_tax_act_fuel_type_id) VALUES (0.1468, '2020-04-01', '2021-03-31',59, 18);
+        INSERT INTO swrs_load.fuel_charge (fuel_charge, start_date, end_date, fuel_carbon_tax_details_id, carbon_tax_act_fuel_type_id) VALUES (0.1919, '2021-04-01', '9999-12-31',59, 18);
+        INSERT INTO swrs_load.fuel_charge (fuel_charge, start_date, end_date, fuel_carbon_tax_details_id, carbon_tax_act_fuel_type_id) VALUES (0.0495, '1899-12-30', '2017-03-31',45, 19);
+        INSERT INTO swrs_load.fuel_charge (fuel_charge, start_date, end_date, fuel_carbon_tax_details_id, carbon_tax_act_fuel_type_id) VALUES (0.0495, '2017-04-01', '2018-03-31',45, 19);
+        INSERT INTO swrs_load.fuel_charge (fuel_charge, start_date, end_date, fuel_carbon_tax_details_id, carbon_tax_act_fuel_type_id) VALUES (0.0578, '2018-04-01', '2019-03-31',45, 19);
+        INSERT INTO swrs_load.fuel_charge (fuel_charge, start_date, end_date, fuel_carbon_tax_details_id, carbon_tax_act_fuel_type_id) VALUES (0.066, '2019-04-01', '2020-03-31',45, 19);
+        INSERT INTO swrs_load.fuel_charge (fuel_charge, start_date, end_date, fuel_carbon_tax_details_id, carbon_tax_act_fuel_type_id) VALUES (0.066, '2020-04-01', '2021-03-31',45, 19);
+        INSERT INTO swrs_load.fuel_charge (fuel_charge, start_date, end_date, fuel_carbon_tax_details_id, carbon_tax_act_fuel_type_id) VALUES (0.0832, '2021-04-01', '9999-12-31',45, 19);
+        INSERT INTO swrs_load.fuel_charge (fuel_charge, start_date, end_date, fuel_carbon_tax_details_id, carbon_tax_act_fuel_type_id) VALUES (30.66, '1899-12-30', '2017-03-31',55, 21);
+        INSERT INTO swrs_load.fuel_charge (fuel_charge, start_date, end_date, fuel_carbon_tax_details_id, carbon_tax_act_fuel_type_id) VALUES (30.66, '2017-04-01', '2018-03-31',55, 21);
+        INSERT INTO swrs_load.fuel_charge (fuel_charge, start_date, end_date, fuel_carbon_tax_details_id, carbon_tax_act_fuel_type_id) VALUES (35.77, '2018-04-01', '2019-03-31',55, 21);
+        INSERT INTO swrs_load.fuel_charge (fuel_charge, start_date, end_date, fuel_carbon_tax_details_id, carbon_tax_act_fuel_type_id) VALUES (40.88, '2019-04-01', '2020-03-31',55, 21);
+        INSERT INTO swrs_load.fuel_charge (fuel_charge, start_date, end_date, fuel_carbon_tax_details_id, carbon_tax_act_fuel_type_id) VALUES (40.88, '2020-04-01', '2021-03-31',55, 21);
+        INSERT INTO swrs_load.fuel_charge (fuel_charge, start_date, end_date, fuel_carbon_tax_details_id, carbon_tax_act_fuel_type_id) VALUES (51.1, '2021-04-01', '9999-12-31',55, 21);
```

Comment:
"Tire - shredded" becomes super confusing with covid: it uses the "combustible waste" rate for columns 10 and 11 of the CTA Schedule 2, but because of the methodology change it was less expensive in 2020, so the rate change was not delayed (so same idea as for Naphta and Coke Oven Gas, let's put a comment here)